### DISCUSSION TITLE
Configure the Google Assistant Unofficial Desktop Client with the computer

### DISCUSSION THREAD

**Discussion by DevDustyPlayer** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#340] [Q&A]
Is there any way to make the Google Assistant Unofficial Desktop Client interact with my computer for things like: Turn off the computer, open an app, make a directory, remove a directory ecc.?

**Comment by Melvin-Abraham** [comment]
This feature is planned for the future possibly in the form of Routines.

**Comment by dahcucumber** [comment]
You can turn computer off etc using assistant computer control on ifttt
